client/src/client/__init__.py

```diff
@@ -30,46 +31,99 @@ def __init__(self) -> None:
         self.log = logging.getLogger("cans-logger")
         self.startup = Startup()
 
+        self.event_loop = asyncio.get_event_loop()
+        self.db_manager = DatabaseManager(
+            name=str(self.startup.db_path),
+        )
+
+        assert StartupState
+        # All of those callbacks should be in the form
+        # function(input: str) -> feedback: str
+        # they can be in a different form (like upstream message)
+        # but then they have to have custom implementation
+        self.ui = UserInterface(
+            loop=self.event_loop,
+            input_callbacks={
```

Comment:
Could we go back to this being a single callback?

Reply:
We can, but this would be easier to extend
At this point it doesn;t matter though, I can make it single

Reply:
Have you thought about support for commands that talk to the outside world, e.g. sharing contacts? If you'd like to use a callback for that, then leave this a dictionary, otherwise (queue or other mechanism), simplify to single callback.